### PR TITLE
Docs to help with DShop's initial set up

### PR DESCRIPTION
**Pull request by phyninja**
As per #992, our repo needs clearer documentation to help new shop admins fill out the 'Server Configuration' page during the onboarding process.

### CODE REVIEW

**Comment by phyninja** [comment]
@mikeshultz
When possible, please take a look at these proposed changes.

Gratefully,
Rajath

**Comment by phyninja** [comment]
@mikeshultz,
Could you take a look at this if you're available, please?

File: docs/dshop/deployment/first-run.md [hunk 1]
```diff
@@ -14,11 +14,35 @@ Enter the super-admin information on the onboarding page.
...
 ![Configuration](images/firstrun-configuration.png)
 
-Congratulations! You have successfully set up the Dshop deployer which allows you (and anyone you give access to) to launch and administer Dshops.
+Use the following field entries as a guide to fill out this page:
+
+- If you are a merchant running a DShop in a **production** environment: 
+  - ```Ethereum Network```: Mainnet
+  - ```Root Domain```: The Domain name associated with your shop.
+  - ```Marketplace Listing ID```: This field has the format <chain_id>-<marketplace_contract_version>-<listing_seq>. Use 1-001-1 to set a _default_ value here.
+  - ```IPFS Gateway``` and ```IPFS API```: These two fields are dependent on the IPFS provider used to host your shop data. For example, if you choose to go with Infura, the fields will take on https://ipfs.infura.io/ and https://ipfs.infura.io:5001/ as values. Contact DShop Support if you have questions.
```
Comment (franckc) [inline docs/dshop/deployment/first-run.md:27]: I'd remove "Contact DShop Support" since Origin no longer provides direct dshop support anymore.

**Review by franckc — COMMENTED** [review]
One minor comment. PTAL

**Review by franckc — APPROVED** [review]
LGTM

**Pull request merged**